test/tls/example_client.py

```diff
@@ -34,6 +40,12 @@
                     help="File to write a ticket to (for TLS 1.3)")
 parser.add_argument("--res_master",
                     help="Resumption master secret (for TLS 1.3)")
+parser.add_argument("--debug", action="store_const", const=5, default=0,
+                    help="Enter debug mode")
+parser.add_argument("server", nargs="?", default="127.0.0.1",
+                    help="The server to connect to")
```

Comment:
Adding an `--sni` argument will be nice. We could specify the IP  address and the TLS Server Name independently.